syne_tune/optimizer/schedulers/searchers/hebo/__init__.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+__all__ = []
+
+try:
+    from syne_tune.optimizer.schedulers.searchers.hebo.hebo_searcher import (  # noqa: F401
+        HEBOSearcher,
+    )
+
+    __all__.extend(
+        [
+            "HEBOSearcher",
+        ]
+    )
+except ImportError as e:
+    logging.debug(e)
```

Comment:
Can you raise a warning instead of debug? Also mention in the warning that uses need to install optuna to use HEBO